include/qfusion.hpp

```diff
@@ -16,6 +16,56 @@
 
 namespace Qrack {
 
+struct BitBuffer;
+typedef std::shared_ptr<BitBuffer> BitBufferPtr;
+typedef std::shared_ptr<complex[4]> BitOp;
+
+// We declare this as a class, rather than a struct, because it should be a nullable type:
```

Comment:
Remove

Reply:
Replaced with a correct and informative comment.